Idref.js

```diff
@@ -53,7 +53,7 @@ function getSearchResults(doc, checkOnly) {
 		// We need to replace the http://www.sudoc.fr/XXXXXX links are they are redirects and aren't handled correctly from subtranslator
 		href = href.replace(/http:\/\/www\.sudoc\.fr\/(.*)$/, "http://www.sudoc.abes.fr/xslt/DB=2.1//SRCH?IKT=12&TRM=$1");
 
-		if ((href.includes("www.sudoc.abes.fr")) || (href.includes("archives-ouvertes")) || (href.includes("catalogue.bnf.fr")) || (href.includes("www.theses.fr")) || (href.includes("pub.orcid.org"))) {
+		if ((href.includes("www.sudoc.abes.fr")) || (href.includes("archives-ouvertes")) || (href.includes("catalogue.bnf.fr")) || (href.includes("www.theses.fr")) || (href.includes("pub.orcid.org")) || (href.includes("www.persee.fr")) || (href.includes("oatao.univ-toulouse.fr"))) {
```

Comment:
This is fine, but I'm wondering if it'd be possible to make sure href is just the host name of the redirected page and then test them against an array along the lines of `["catalogue.bnf.fr", "www.sudoc.abes.fr"...].includes(href)`

This would perform much better and imo be a lot more legible, but given that we're still testing against a relatively short list of strings, the current version is fine -- just let me know if you think the above is viable and you're interested in working on it or not.